docker/Dockerfile-windows-base

```diff
@@ -1,9 +1,16 @@
 ARG WINDOWS_VERSION=1809
 #This version confirmed to work for both github and AKS
 FROM mcr.microsoft.com/windows/servercore:$WINDOWS_VERSION as servercore
-FROM mcr.microsoft.com/windows/nanoserver:$WINDOWS_VERSION
+
 # Nanoserver is missing exactly one dll that go needs. The servercore image has this dll, but
 # is nearly 2GB. This has the potential to be somewhat fragile, but it is the only way I can
 # find to keep the container image size small. To keep from doing this rather expensive
 # multistage build every time, we build a small base image and push it to our repo
+FROM mcr.microsoft.com/windows/nanoserver:$WINDOWS_VERSION
```

Comment:
I was not able to find a reference to this Dockerfile in our GitHub workflows. Is there a process in place to update this image?

Reply:
https://github.com/dapr/dapr/actions/workflows/dapr-base-containers.yaml

Reply:
Ah, thanks!